Flatiron - Tuples chapter.md

```diff
@@ -0,0 +1,115 @@
+
+##What's a Tuple?
+Tuples are a very specialized way of grouping and organizing data, and to best understand their value, limitations and purpose we'll compare and contrast them to Arrays, Dictionaries and Classes.
+
+**Arrays** help you manage variably-sized ordered collections of similar data, such as a list of numbers or strings.  For example, we can make an array of some key characters from the movie Star Wars:
+```
+// Array
+let list = [ "Darth", "Ben", "Luke" ]
+print( list[1] )
+```
+Arrays are pretty useful but they're somewhat limited in that their contents can *only* be accessed via indices. 
+
+**Dictionaries** (as you might imagine by the name) help you manage data by mapping bits of it to keys, just like a language dictionary maps definitions to words.  The examples below use strings and arrays of strings but you could use almost anything (subject to certain rules that we'll cover in a later session).
+```
+// Dictionary
+let simpleDictionary = [ "Darth":"big baddie", "Ben":"mentor", "Luke":"protagonist" ]
+let richerDictionary = [
+	"Darth":[ "big baddie", "father of Luke" ],
+	"Ben":[ "mentor", "trainer of Darth" ],
+	"Luke":[ "protagonist", "son of Darth" ]
+	]
+print( richerDictionary[ "Darth" ] )
+[ "big baddie", "father of Luke" ]
+print( richerDictionary[ "Darth" ]![1] )
+"father of Luke"
+```
```

Comment:
Similar to my above comment with regards to adding the swift keyword and cleaning up the spacing.

NOTE: You aren't required to make any changes here, just adding comments to reflect what it would be like if this was an actual lesson.